s/5_3.ss

```diff
@@ -661,6 +665,13 @@
                              (floatable? (imag-part x)))]
          [else #t])))
 
+(define fixnum-floatable-wlop?
+  ;; floatable without loss of precision
+  (lambda (x)
+    (if (<= (fixnum-width) (constant float-precision-bits))
```

Comment:
Let's assume `(fixnum-width)` is equal to `24`. Then fixnums are in the range [-2^23, 2^23 - 1].

Let's assume `(constant float-precision-bits)` is equal to `23`. Then integers in the range [-2^23, 2^23] can be stored in the mantissa of a float without loss of precision.

Therefore, I would replace the line 671 of the file s/5_3.ss with:

    (if (<= (- (fixnum-width) 1) (constant float-precision-bits))

The width of a fixnum includes the sign bit, whereas the width of the mantissa of a float excludes it.

Also, I have suggestions to improve the names of two variables:

* Replace the name of the constant `float-precision-bits` with the name `float-mantissa-width`, because it is not clear what "precision" means (for instance, the precision of a float could be 64 in the case of IEEE 754 binary64).
* Replace the name of the procedure `fixnum-floatable-wlop?` with the name `fixnum-fits-in-mantissa?`. Even though there is an edge case in which this new name is misleading (the integer 2^53 is 54 bits wide, and therefore does not fit in a 53 bits mantissa), I think it is a better name because this procedure rejects a lot of fixnums that **are** floatable without loss of precision, for instance 2^55:

      > (= (expt 2 55) (exact (inexact (expt 2 55))))
      #t